src/js/pyodide.js

```diff
@@ -1,16 +1,16 @@
 /**
  * The main bootstrap code for loading pyodide.
  */
-import { Module } from "./module";
+import { Module } from "./module.js";
```

Comment:
Otherwise I was getting errors like,
```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/rth/src/pyodide/src/js/module' imported from
 /home/rth/src/pyodide/src/js/api.js                                                              
Did you mean to import ../module.js?
```